modules/microshift-custom-ca-cert-cleaning.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-certificates-cleaning_{context}"]
+= Cleaning up and recreating the custom certificates
+
+To stop the {microshift-short} services, clean up the {microshift-short} certificates and recreate the {microshift-short} certificates, use the following steps.
```

Comment:
The title refers to cleaning up and recreating the custom certificates, but this lead sentence refers specifically to Microshift services and certificates (and addresses stopping) as well. 
Is it possible to make the title and lead sentence more in line with one another? 
Don't know enough about the feature but wanted to point that out regardless.

Reply:
I have replaced "Microshift" with "custom". This should be good.